### ISSUE TITLE
Missing projects list route

### ISSUE THREAD

**Issue by Allcharles** [QutEcoacoustics/baw-server#411]
# Missing Route

Associated Page: https://staging.ecosounds.org/projects/new_access_request
Expected Route: ???
Expected Method: GET
Expected Body: 

- token

Expected Action: Should provide a list of projects the user can request access to

**Comment by peichins** [comment]
Do we need this page at all? @atruskie

**Comment by atruskie** [comment]
Yeah, we do. I don't want us to be gate-keepers. Ideally, users should be managing this stuff all by themselves.

Probable new route: `/projects/inaccessible`
